polygeist/tools/cgeist/Lib/ConstantFolder.cc

```diff
@@ -0,0 +1,21 @@
+//===- ConstantFolder.cc -----------------------------------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "ConstantFolder.h"
+
+using namespace mlir;
+using namespace mlirclang;
+
+Value mlirclang::ConstantFolder::foldFPCast(Location Loc, Type PromotionType,
```

Comment:
There is a using declaration at line 12, no need to qualify with `mlirclang` here.